website/docs/components/form/file-input/partials/specifications/states.md

```diff
@@ -0,0 +1,7 @@
+## States
+
+![Example of file input states](/assets/components/form/file-input/file-input-states.png)
+
+### Filled states 
+
+![Example of file input filled states](/assets/components/form/file-input/file-input-filled-states.png)
```

Comment:
I added an alternative visual approach to this section into the Figma website assets file and tagged you in some comments.